doctest/build.gradle

```diff
@@ -34,6 +34,22 @@ task bootstrap(type: Exec, dependsOn: ['cloneSqlCli']) {
 
 task startPrometheus(type: SpawnProcessTask) {
     doFirst {
+        def pidFile = new File(path, ".prom.pid.lock")
+        if (!pidFile.exists()) {
+            logger.quiet "No Prometheus server running!"
+        } else {
+            def pid = pidFile.text
+            def process = "kill $pid".execute()
+
+            try {
+                process.waitFor()
```

Comment:
Do we not want to check result?

Normally, this is crucial when spawning other processes. Is "we tried, I hope it worked" sufficient in this case?

Reply:
What should happen if it was not successful? I think now it'll produce an error.